e2e-tests/playwright/utils/tekton/tekton.ts

```diff
@@ -0,0 +1,67 @@
+import { expect, Page } from '@playwright/test';
+import { UIhelper } from '../UIhelper';
+
+export class Tekton {
+  page: Page;
+  uiHelper: UIhelper;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.uiHelper = new UIhelper(page);
+  }
+
+  getAllGridColumnsTextForPipelineRunsTable() {
+    return ['NAME', 'STATUS', 'TASK STATUS', 'STARTED', 'DURATION'];
+  }
+
+  async clickOnExpandRowFromPipelineRunsTable() {
+    await this.page
+      .locator(
+        'table[aria-labelledby="Pipeline Runs"] button[aria-label="expand row"]',
+      )
+      .first()
+      .click();
+  }
+
+  async openModalEchoHelloWorld() {
+    const loc = 'g[data-test="task echo-hello-world"]';
+    await this.page.waitForSelector(loc);
+    await this.page.locator(loc).click();
+  }
+
+  async verifyTableHasRows() {
+    const rowCount = await this.page
+      .locator('table[aria-labelledby="Pipeline Runs"] tbody tr')
```

Comment:
can we move this locator to pageObjects.

Reply:
@subhashkhileri this locator is very specific for this page